src/device-registry/utils/date.js

```diff
@@ -164,14 +162,14 @@ function threeMonthsFromNow(date) {
 }
 
 module.exports = {
-  generateDateFormat,
-  threeMonthsFromNow,
-  generateDateFormatWithoutHrs,
-  addMonthsToProvideDateTime,
-  monthsInfront,
-  isTimeEmpty,
-  getDifferenceInMonths,
-  addDays,
-  addMinutes,
-  formatDate,
+  generateDateFormat: generateDateFormat,
```

Comment:
this is just small small stuff, I do not think it had/has any significant effect.

Reply:
This is fine @Baalmart .
I just find it a little repeatitive

Reply:
I agree @123MwanjeMike , I guess we can revery this unnecessary check. @BenjaminSsempala

Reply:
@BenjaminSsempala , please help me revert this useless changes of mine and then we proceed with merging this PR. Thanks for the amazing work!

Reply:
Thank You @Baalmart and @123MwanjeMike. I've reverted these now.

Reply:
Thanks @BenjaminSsempala